### DISCUSSION TITLE
Welcome phylocom users!

### DISCUSSION THREAD

**Discussion by camwebb** [phylocom/phylocom#39] [Announcements]
## 👋 Welcome!

We’re using Github's Discussions as a place to connect with other phylocom users. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Be civil and open-minded.

**Please use Discussions for getting help, and Issues for bug reports and feature requests**

**Comment by nawalshrestha** [comment]
I am a new user of this amazing software. I used it successfully with a smaller dataset (tree with <1000 tips). However, when I run the nodesig command on a large dataset (tree with about 17,000 tips), I receive the following error. 

Numerical Recipes run-time error...
allocation failure 2 in matrix()
...now exiting to system...

Is it the limitation of the software or did I miss anything on my part? For your information, I ran the command in a window system with 64 GB RAM.

**Reply by camwebb** [reply]
The size of the tree that leads to this error is unpredictable and seems to depend on a combination of hardware, OS and compiler. The software has not been optimized for large trees - there has been little development of the code in the last 15 years. So I don't think you missed anything. Newick output size is even more limited than Newick input size, as the Newick writer is recursive and hits max stack depth with only a few thousand terminal taxa. Sorry. Lots of work to be done as trees get bigger and bigger!  You may find that the [`R/picante`](https://github.com/skembel/picante) version works better for you.
